docs/policy_configuration.md

```diff
@@ -0,0 +1,160 @@
+      sriov-network-vlan-2: "222"
+```
+**Note:** Only policy configuration values that are exposed in the associated ClusterTemplate under `spec.templateParameterSchema > policyTemplateParameters` can be updated through the ProvisioningRequest.
+
+Once the update is made, the `<cluster-name>-pg` ConfigMap in the `ztp-<cluster-template-namespace>` namespace gets updated with the new value. This ConfigMap is used by the ACM policies in their hub templates.
+```yaml
+$  oc get clustertemplate -A
+NAMESPACE                 NAME                   AGE
+clustertemplate-a-v4-16   clustertemplate-a.v1   3d23h
+
+$  oc get cm -n ztp-clustertemplate-a-v4-16 sno-1-pg -oyaml
+apiVersion: v1
+data:
+  cpu-isolated: 0-1,64-65
+  cpu-reserved: 2-10
+  hugepages-count: "32"
+  hugepages-default: 1G
+  hugepages-size: 1G
+  install-plan-approval: Automatic
+  sriov-network-vlan-1: "111"
+  sriov-network-vlan-2: "222"
+kind: ConfigMap
+metadata:
+  name: cnfdg11-pg
+  namespace: ztp-clustertemplate-a-v4-16
+```
+
+Once a policy matched with a ManagedCluster deployed through a ProvisioningRequest becomes `NonCompliant`, it's reflected in the ProvisioningRequest `status.policies` and the time when it becomes `NonCompliant` is also recorded. The `ConfigurationApplied` condition reflects that the configuration is being applied.
+```yaml
+status:
+  clusterDetails:
+    clusterProvisionStartedAt: "2024-10-07T17:59:23Z"
+    name: cnfdg11
+    nonCompliantAt: "2024-10-07T21:53:29Z"  <<< non compliance timestamp recorded here
+    ztpStatus: ZTP Done
+  conditions:
+  - lastTransitionTime: "2024-10-07T21:53:29Z"
+    message: The configuration is still being applied
+    reason: InProgress
+    status: "False"
+    type: ConfigurationApplied
+  policies:
+  - compliant: Compliant
+    policyName: v1-perf-configuration-policy
+    policyNamespace: ztp-clustertemplate-a-v4-16
+    remediationAction: enforce
+  - compliant: NonCompliant <<< Policy is NonCompliant
+    policyName: v1-sriov-configuration-policy
+    policyNamespace: ztp-clustertemplate-a-v4-16
+    remediationAction: enforce
+  - compliant: Compliant
+    policyName: v1-subscriptions-policy
+    policyNamespace: ztp-clustertemplate-a-v4-16
+    remediationAction: enforce
+```
+
+**Notes**:
+* The format of the way the `nonCompliantAt` timestamps might move to another structure in the status, but it will still be recorded.
+* Some changes happen so fast that the Policy doesn't even switch to `NonCompliant`, so the IMS operator cannot record the event. In this case, IMS still holds a correct recording since all the policies are/remain Compliant.
+* Once an enforce `NonCompliant` Policy becomes `Compliant` again, the `status.policies` is updated, the `status.clusterDetails.nonCompliantAt` value removed and the `ConfigurationApplied` condition updated to show that the configuration is up to date:
+* When refactored, the start and end times of the configuration being NonCompliant will be recorded.
+```yaml
```

Comment:
Add a sentence describing what the following yaml is about - is it an example of the status after the policies have all been applied?